Primo.js

```diff
@@ -274,6 +281,7 @@ function importPNX(text) {
 		if(creators[i]) {
 			var creator  = ZU.unescapeHTML(creators[i].textContent).split(/\s*;\s*/);
 			for(j in creator){
+				creator[j] = creator[j].replace(/\([\d\sabcefilr.? ]*[-–][\d\abcefilr.? ]*\)/gi, '');
```

Comment:
I thought I mentioned this previously, but I don't see it now. That regexp is very obscure. You need to comment on what you're doing here. Chances are you don't want to be replacing random letters, but actual words.